app/lib/three_scale/analytics/user_tracking.rb

```diff
@@ -31,11 +31,19 @@ def method_missing(method_sym, *args, &block)
         end
 
         def segment_configured?
```

Comment:
As CodeClimate says, this can become a class method instead of being an instance method 😄 Anyway, not your fault 👍